New package: Extrae v0.2.0

JuliaRegistrator opened the pull request:
- Registering package: Extrae
- Repository: https://github.com/bsc-quantic/Extrae.jl
- Created by: @mofeing
- Version: v0.2.0
- Commit: 962fe1eadc84abc618aed5b4b359dccf4e9a012a
- Git reference: HEAD
- Description: Julia bindings for extrae Basic API

goerz commented:
Can you extend the README just a little bit to give some context of what this is for? What exactly is the "`extrae` Basic API"? Can you make that a link? Is this wrapping https://github.com/bsc-performance-tools/extrae? (took me a few tries to google).

I'm sure this package is for a very narrow audience and that audience probably knows exactly what this is about. Still, it would be nice if there was a little more information for someone randomly coming across the package.

[noblock]

mofeing commented:
Done! What do you think now?

[noblock]

JuliaHubRegistrator committed:
New package: Extrae v0.2.0

UUID: 8a0c07fa-ade5-4b2a-b81a-b192b2bedf88
Repo: https://github.com/bsc-quantic/Extrae.jl.git
Tree: 82a6afd27c4b6a467fe860ae951a909a9f6543e7

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
